src/main/java/com/jakewharton/retrofit2/adapter/rxjava2/RxJava2CallAdapter.java

```diff
@@ -69,13 +69,13 @@
       return observable.toFlowable(BackpressureStrategy.LATEST);
     }
     if (isSingle) {
-      return observable.toSingle();
+      return observable.single(null);
```

Comment:
Thanks that does work though it's quite a waste to first convert it to Maybe and then to Single